src/cli/encrypt.go

```diff
@@ -76,8 +77,23 @@ func encryptCommand(root *rootCommand) *cobra.Command {
 				logger.Info("Dry run, nothing changed.")
 				return nil
 			}
+			if len(unencryptedGroups) == 0 {
+				return nil
+			}
```

Comment:
ak niesu ziadne hodnoty na zasifrovanie tak chcem aby sa len vypisalo `No values to encrypt` bez `Encrypt done.` ktory sa vypisuje na konci potom co prebehne encryption.

Reply:
Ok. Ale potom je potrebne upravit aj ostatne `cmds` aby sa spravali rovnako.

Na vypise (vid dole) nevidim nic zle, proste prikaz uspesne skoncil.
Kedze je pred tym informacia o `dry-run`, tak si nemyslim, ze by to niekto pochopil nespravne.
```
Dry run, nothing changed.
Pull/push/encrypt done.
```

TLDR: je mi to jedno, len nech to je prosim vsade rovnako ;)

Reply:
j suhlas ze to ma byt konzistentne, ak to ostatne cmds robia inak tak tu to upravim podla toho.